frontend/src/features/public-form/components/FormFields/FieldFactory.tsx

```diff
@@ -38,68 +38,69 @@ import {
 
 interface FieldFactoryProps {
   field: FormFieldWithQuestionNo
+  isPrefilled?: boolean
   colorTheme?: FormColorTheme
 }
 
 export const FieldFactory = memo(
-  ({ field, colorTheme }: FieldFactoryProps) => {
+  ({ field, ...rest }: FieldFactoryProps) => {
```

Comment:
Just checking: does this syntax mean typescript will only inject `isPrefilled` and `colorTheme` into `rest`, or will any other supplied props also go through (e.g. `key`, which was previously ignored)? Assuming they do go through, any issue with letting `key` propagate?